### PR TITLE
Update LIT5129Qene.xml

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Some revisions in a qene work record.

### CODE REVIEW

File: 5001-6000/LIT5129Qene.xml [hunk 1]
```diff
@@ -5,7 +5,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
         <fileDesc>
             <titleStmt>
                 <title xml:lang="gez" xml:id="t1">በምጽአተ፡ ክርስቶስ፡ ስንቃዊ፡</title>
-                <title xml:lang="gez" type="normalized" corresp="#t1">Qǝne 'Ba-mǝṣʾata krǝstos sǝnqāwi'</title>
+                <title xml:lang="gez" type="normalized" corresp="#t1">Ba-mǝṣʾata krǝstos sǝnqāwi ... (Qǝne of kǝbr yǝʾǝti type)</title>
+                <title xml:lang="en" type="normalized" corresp="#t1">Through the advent of Christ, the musucian, ...</title>
```
Comment (CarstenHoffmannMarburg) [inline 5001-6000/LIT5129Qene.xml:9]: musician

File: 5001-6000/LIT5129Qene.xml [hunk 2]
```diff
@@ -52,15 +55,31 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                  <l n="1">በምጽአተ፡ ክርስቶስ፡ ስንቃዊ፡</l>
+                  <l n="2">በከናፍረ፡ ብእሲት፡ ዘማ፡ ወበመሥገርትየ፡ አፈ፡ ነዓዊ፡</l>
+                  <l n="3">ኦካህን፡ ትምክህተ፡ ቤቱ፡ ለሌዊ፡</l>
+                  <l n="4">ለምንት፡ ሞትከ፡ እምትሙት፡ ዘማዊ።</l>
               </ab>
+                </div>
+                
+                <div type="translation" xml:lang="en" xml:id="translationEthio" corresp="#etiopica">                    
+                        <ab>While many murmured for the fact that the appointment
+                            <l n="1">Through the advent of Christ, the musucian,</l>
```
Comment (CarstenHoffmannMarburg) [inline 5001-6000/LIT5129Qene.xml:77]: musician

**Review by CarstenHoffmannMarburg — CHANGES_REQUESTED** [review]

File: 5001-6000/LIT5129Qene.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply 5001-6000/LIT5129Qene.xml:9]: ok

File: 5001-6000/LIT5129Qene.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply 5001-6000/LIT5129Qene.xml:77]: ok

**Review by CarstenHoffmannMarburg — APPROVED** [review]

**Pull request merged**